devops/containers/ubuntu2404_base.Dockerfile

```diff
@@ -8,6 +8,10 @@ USER root
 COPY scripts/install_build_tools.sh /install.sh
 RUN /install.sh
 
+# Build zstd from sources with -fPIC flag.
```

Comment:
could we maybe add a link to the ubuntu bug report so we can check to remove this in the future?

Reply:
I've added the link to intel/llvm bug report instead: https://github.com/intel/llvm/issues/15935
I'll likely have to open another bug report to Debian instead (as suggested here: https://bugs.launchpad.net/ubuntu/+source/libzstd/+bug/2086543/comments/3).